site/docs/skylark/deploying.md

```diff
@@ -63,6 +62,24 @@ We would have the following structure:
     test.mocs
 ```
 
+### WORKSPACE
+
+In the project's `WORKSPACE`, you should define the name that users will use
+to reference you rules. If your rules belong to the
+[bazelbuild](https://github.com/bazelbuild) organization, you must use
+`rules_<lang>` (e.g. `rules_mockascript`). Otherwise, you should name your
+repository `<org>_rules_<lang>` (e.g. `build_stack_rules_proto`). Please contact
+[bazel-dev mailing list](https://groups.google.com/forum/#!forum/bazel-dev)
+if you feel like your rules should follow the convention for rules in the
+[bazelbuild](https://github.com/bazelbuild) organization.
```

Comment:
My understanding is that the name is an implementation detail of a repository (ideally, they shouldn't rely on it). So I'd rather not have users send us an email for this.

Maybe just remove this sentence?